ceph-build/config/definitions/ceph-build.yml

```diff
@@ -23,16 +23,15 @@
             - x86_64
             - arm64
       - axis:
+          # the dynamic axis is implemented **oddly** in jenkins job builder.
+          # It will pick just the first item in `values` and assign that to the
+          # 'variable name' that is where `DIST` would get expanded too. This
+          # is not documented. This means that DIST_VERSIONS will need to exist
+          # as an environment variable passed onto this job.
```

Comment:
since we've already got a comment, let's elaborate:  "..containing a space-separated list of all values to substitute for DIST in this job"